### ISSUE TITLE
Split solution into individual webparts/extensions

### ISSUE THREAD

**Issue by absieee** [SharePoint/sp-starter-kit#121]
#### Category
- [x] Question
- [ ] Bug
- [ ] Enhancement

Is it possible to split this solution into individual webparts and extensions? So for example, instead of uploading the entire kit as a packaged solution and making all the webparts and extensions available for a site collection, say if I want to just make specific webparts available for use, then would that be possible to do? If so, how do I go about doing so?

Thanks.

**Comment by elpinto007** [comment]
HI Abhishek

I actually am doing this right now. I have been splitting out some of the web parts and extensions so I can individually deploy them to my tenant.
I have the Tiles web part and the Alerts extension working so far (available from my repositories).
The web parts are pretty simple to do, I simply created a new project for a web part, and brought over the code from the web part I am interested in.
The tricky part is to ensure the correct dependencies are also installed/available.
I am currently working on the footer for the communication site.

Regards,
Krishant Lodhia
Sent from my tablet

________________________________

**Comment by absieee** [comment]
Hi Krishant,

I figured out how to break them up shortly after posting that question. Got the tiles, banner and clocks webparts working individually right now. Will post them up as individual repos once I do the same with all the webparts and extensions.

**Comment by elpinto007** [comment]
Awesome stuff!

I'm actually stuck on the footer. The office react fabric ui has reworked how the command bar works so it's causing all sorts of errors for me. Let me know if you get this to work.

Krishant Lodhia
Sent from my mobile device
________________________________

**Comment by memsim1010** [comment]
@absieee & @elpinto007 Have you had any luck with the People directory, and personal web parts?  Your repos are great and I'd love to see more!

**Comment by elpinto007** [comment]
> @absieee & @elpinto007 Have you had any luck with the People directory, and personal web parts? Your repos are great and I'd love to see more!

Hi! I just came across this. Sorry for the delay. But yes! I have managed to get the Employee Directory working for myself. I havent uploaded it to my repository yet. Let me know if you were still looking for this.
Thanks!

**Comment by mickyc1** [comment]
Hi @elpinto007 , would be great if you could upload the isolated Employee directory!

**Comment by elpinto007** [comment]
> Hi @elpinto007 , would be great if you could upload the isolated Employee directory!

Hi @mickyc1! Absolutely. I will upload it to my repository later today.

**Comment by mickyc1** [comment]
Thanks @elpinto007, please let me know when this is done :)

**Comment by elpinto007** [comment]
@mickyc1 , try [this](https://github.com/elpinto007/employee-directory). I was having some difficulties uploading the repository to Github. 
I also realised that I didnt create my original Employee Directory from the latest commit in SharePoint PNP. But the general functionality is still the same.
I added in 2 filters in the Search query that retrieves the list of employees, one checks if the account has an email address, and the other if the account is not hidden from GAL (usually for employees that have left but their accounts are still active).
